pkg/transforms/outputdata.go

```diff
@@ -0,0 +1,65 @@
+//
+// Copyright (c) 2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package transforms
+
+import (
+	"encoding/json"
+	"errors"
+
+	"github.com/edgexfoundry/app-functions-sdk-go/appcontext"
+)
+
+// OutputData houses transform for outputting data to configured trigger response, i.e. message bus
+type OutputData struct {
+}
+
+func (f OutputData) SetOutputData(edgexcontext *appcontext.Context, params ...interface{}) (bool, interface{}) {
+
+	edgexcontext.LoggingClient.Debug("Setting output data")
+
+	if len(params) < 1 {
+		// We didn't receive a result
+		return false, nil
+	}
+
+	var data []byte
+	var err error
+
+	switch params[0].(type) {
+	case string:
+		input := params[0].(string)
+		data = []byte(input)
+
+	case []byte:
+		data = params[0].([]byte)
+
+	case json.Marshaler:
```

Comment:
Do we want to add an issue to do this same input data handling in export functions or add that to this PR?